sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -3,6 +3,7 @@
 
 package com.azure.cosmos.encryption;
 
+import brave.sampler.Matchers;
```

Comment:
Where is this `brave` library coming from?

Reply:
Added by mistake, thanks for pointing out... removed the import